### PR TITLE
Meteor/18

### PR DESCRIPTION
**Pull request by sebakerckhof**
Wait on 1.8 to finish & merge

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ Package.describe({
 
 Package.registerBuildPlugin({
   name: "compileScssBatch",
-  use: ['caching-compiler@1.1.9', 'ecmascript@0.9.0'],
+  use: ['caching-compiler@1.2.0-beta171.4', 'ecmascript@0.12.0-beta171.4'],
```
Comment (macrozone) [inline package.js]: should probably use the release version of these plugins before merge

File: package.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ Package.registerBuildPlugin({
 });
 
 Package.onUse(function (api) {
-  api.versionsFrom("1.6.1.1");
+  api.versionsFrom("1.7.1-beta.4");
```
Comment (macrozone) [inline package.js]: same here

**Comment by macrozone** [comment]
@sebakerckhof can you do an atmosphere release of that?

**Comment by sebakerckhof** [comment]
@macrozone yep, released as version `4.10.0`

**Pull request merged**